rel/config.exs

```diff
@@ -52,10 +64,24 @@ release :mana do
     blockchain: :permanent,
     cli: :permanent,
     evm: :permanent,
-    ex_wire: :permanent,
+    ex_wire: :load,
```

Comment:
I don't think load is needed.

Reply:
When I had load permanent before, I think it started the application before I had a chance to configure it.

Reply:
looks like you're right, doesn't start up without starting it ourselves even if it's permanent. fixed.